src/mlpack/methods/kde/kde.hpp

```diff
@@ -214,6 +219,37 @@ class KDE
   //! Modify the mode of KDE.
   KDEMode& Mode() { return mode; }
 
+  //! Get whether Monte Carlo estimations are being used or not.
+  bool MonteCarlo() const { return monteCarlo; }
+
+  //! Modify whether Monte Carlo estimations are being used or not.
+  bool& MonteCarlo() { return monteCarlo; }
+
+  //! Get Monte Carlo probability of error being bounded by relative error.
+  double MCProbability() const { return MCProb; }
+
+  //! Modify Monte Carlo probability of error being bounded by relative error.
+  //! (0 <= newProb < 1).
+  void MCProbability(const double newProb);
+
+  //! Get Monte Carlo initial sample size.
+  size_t MCInitialSampleSize() const { return initialSampleSize; }
+
+  //! Modify Monte Carlo initial sample size.
+  size_t& MCInitialSampleSize() { return initialSampleSize; }
+
+  //! Get Monte Carlo entry coefficient.
+  double MCEntryCoefficient() const { return MCEntryCoef; }
+
+  //! Modify Monte Carlo entry coefficient. (newCoef >= 1).
+  void MCEntryCoefficient(const double newCoef);
+
+  //! Get Monte Carlo break coefficient.
+  double MCBreakCoefficient() const { return MCBreakCoef; }
```

Comment:
Once you change the names of the parameters, then the names of the functions here can exactly match the name of the variable (i.e. `MCBreakCoef()` for `mcBreakCoef`).